bootstrap.d/dev-python.y4.yml

```diff
@@ -0,0 +1,14 @@
+tools:
+  - name: host-bragi
+    labels: [aarch64, riscv64]
+    architecture: noarch
+    from_source: bragi
+    tools_required:
+      - host-python
+    revision: 1
+    install:
+      - args: ['python3', '-m', 'venv', '.']
+        workdir: '@PREFIX@'
+      - args: ['bin/pip', 'install', '-e', '@THIS_SOURCE_DIR@']
```

Comment:
`-e` doesn't work (the source dir will not be present when downloading the package from xbbs etc).